sig-architecture/production-readiness.md

```diff
@@ -0,0 +1,51 @@
+# Production Readiness Review Process
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable, can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production.
+
+## Status
+
+The process and questoinnaire are currently under development as part of the
+[PRR KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/20190731-production-readiness-review-process.md), with a target that reviews will be needed for features
+going into 1.18.
+
+During the 1.17 cycle, the PRR team will be piloting the questionnaire and other
+aspects of the process.
+
+## Questionnaire
+
+* Feature enablement and rollback
+  - How can this feature be enabled / disabled in a live cluster?
+  - Can the feature be disabled once it has been enabled (i.e., can we roll
```

Comment:
It might be good to include impact on workloads as well, distinct from the control-plane/cluster-level considerations.
Like, some workload considerations might be:
- Does this feature change the behavior or performance characteristics of workloads running on a cluster?
- Will some workloads that could run successfully on the cluster before, stop working or no longer be admissible once this feature is enabled?
- Do workloads need to be restarted to take advantage of this feature? 
- How can workloads be migrated over to take advantage of this feature? Can it be selectively enabled (e.g. per-node/per-namespace, only to new workloads/objects, in a report-only or dry-run mode)? Will enabling/disabling the feature require downtime or make certain features temporarily unavailable for workloads running on the cluster?